kart/working_copy/base.py

```diff
@@ -1010,6 +1046,18 @@ def _delete_meta(self, sess, dataset):
         """
         raise NotImplementedError()
 
+    def rewrite_full(self, commit, *datasets, force=False):
```

Comment:
should this method be done in one transaction? Otherwise a client watching carefully will see tables be dropped, and then recreated shortly afterwards. For big WCs those could be a while apart.

Reply:
Done